migration-rules/v0.9.28/rules/src/main/scala/fix/v0_9_28.scala

```diff
@@ -15,6 +15,13 @@ class v0_9_28 extends SemanticRule("v0_9_28") {
       case t @ init"SemanticRuleSuite(..$_)" =>
         Patch.addGlobalImport(importer"org.scalatest.FunSuiteLike") +
           Patch.replaceTree(t, "AbstractSemanticRuleSuite with FunSuiteLike")
+
+      case t @ init"SemanticRuleSuite" =>
+        Patch.addGlobalImport(importer"org.scalatest.FunSuiteLike") +
+          Patch.replaceTree(t, "AbstractSemanticRuleSuite with FunSuiteLike")
```

Comment:
Did you consider https://github.com/scalacenter/scalafix/pull/1395#discussion_r631361416 to avoid several matching like this. Also, I guess this is untested?

Reply:
it is tested. the new case doesn't have `()` after SemanticRuleSuite, so the previous pattern match wasn't working. 
and for the comment, it doens't erase the other traits (which is maybe weird): the second example `Case2` mixin another trait, and it's not erased